components/TalkCard/TalkCard.js

```diff
@@ -0,0 +1,41 @@
+//import { data } from "../utils/data";
+import styled from "styled-components";
+import { data } from "../utils/data";
+
+export default function TalkCard() {
+  return (
+    <>
+      <h2>JURASSIC TALK 2022 Schedule</h2>
+      <StyledUl>
+        {data?.map((talk) => (
+          <StyledLi key={talk.id}>
+            <h4>{talk.title}</h4>
+            <h4>{talk.authors[0]}</h4>
```

Comment:
Please use descening headline tags. Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de) about it.